### PR TITLE
 #160617641 implement liking and disliking an article

### PR DESCRIPTION
**Pull request by nakiwuge**
#### What does this PR do?
- Enables users to like and dislike an article
#### Description of Task to be completed?
- Enable authenticated users to like or dislike articles
- Enable users to update their choices
#### How should this be manually tested?
- When you click on an article from the home page, you will be redirected to view that article. 
- Go to the sidebar Click thumbs up to like or thumbs down to dislike
#### What are the relevant pivotal tracker stories?
[#160617641] https://www.pivotaltracker.com/n/projects/2198902/stories/160617641
#### Screenshots 
screen shot 2018-11-19 at 21 15 06

### CODE REVIEW

File: src/actions/actionTypes.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ export const PASSWORD_RESET= "PASSWORD_RESET";
 export const NEW_PASSWORD="NEW_PASSWORD";
 export const USER_NOT_FOUND="USER_NOT_FOUND";
 export const REGISTER_USER= "REGISTER_USER";
+export const LIKE = "LIKE";
+export const DISLIKE = "DISLIKE";
+
 const ACTION_TYPE = {
   NEW_ARTICLE: "NEW_ARTICLE",
   VIEW_ARTICLES: "VIEW_ARTICLES",
```
Comment (LehruAngela) [inline src/actions/actionTypes.js]: Add your actions to the ACTION_TYPE dictionary, and use the exported ACTION_TYPE to call your actions, so as to have code consistency

**Review by LehruAngela — CHANGES_REQUESTED** [review]

File: src/actions/actionTypes.js (hunk 1 above)
Comment (nakiwuge) [reply src/actions/actionTypes.js]: @LehruAngela i have made this change

**Comment by LehruAngela** [comment]
@zellymiriam rebase with develop

**Comment by LehruAngela** [comment]
@zellymiriam rebase with develop

**Pull request merged**